Live: Experimental HA with Redis (#36787)
grafana/grafana: docs/sources/administration/configuration.md

allowed_origins = "https://*.example.com"
```

### ha_engine

> **Note**: Available in Grafana v8.1 and later versions.

**Experimental**

> The name of high-availability engine for Grafana Live. By default, it's not set. The only possible value is "redis".

Reviewer: ```suggestion
The high availability (HA) engine name for Grafana Live. By default, it's not set. The only possible value is "redis".
```

> More information in [Configure Grafana Live HA setup]({{< relref "../live/live-ha-setup.md" >}}).

Reviewer: ```suggestion
For more information, refer to [Configure Grafana Live HA setup]({{< relref "../live/live-ha-setup.md" >}}).
```

### ha_engine_address

> **Note**: Available in Grafana v8.1 and later versions.

**Experimental**

> Address string of selected HA Live engine. For Redis it's a `host:port` string. Example:

Reviewer: ```suggestion
Address string of selected the high availability (HA) Live engine. For Redis, it's a `host:port` string. Example:
```
